docs/usage.md

```diff
@@ -22,6 +22,10 @@ This samplesheet is then specified on the command line as follows:
 --input '[path to samplesheet file]' --databases '[path to database sheet file]'
 ```
 
+Note pipeline supports both CSV and PEP input sample sheets. Find out more [here](http://pep.databio.org/en/2.1.0/specification/).
+When using PEP as an input, the `samplesheet.csv` must be placed in the same folder
+as `config.yaml` file. A path to `samplesheet.csv` within the config must be absolute.
```

Comment:
```suggestion
> ⚠️ When using PEP as an input, the `samplesheet.csv` must be placed in the same folder
> as `config.yaml` file. A path to `samplesheet.csv` within the config must be absolute.
```

Must it be absolute? I thought what we talked about before was that it should be relative to input_base_dir?

Reply:
I had some problems with that. What Nextflow does when we pass `pep_input_base_dir` is creating workdir with specific files structure. For the command:
```
nextflow run main.nf -profile test_pep,docker --outdir /home/cgf8xr/nextflow-output/ --input ~/Downloads/test_pep/config.yaml
```
The workdir content looks as follows:
```
(...)
-rw-rw-r-- 1 cgf8xr cgf8xr   298 Sep 19 10:24 .command.sh
lrwxrwxrwx 1 cgf8xr cgf8xr    43 Sep 19 10:24 config.yaml -> /home/cgf8xr/Downloads/test_pep/config.yaml
lrwxrwxrwx 1 cgf8xr cgf8xr    69 Sep 19 10:24 samplesheet_schema.yaml -> /home/cgf8xr/databio/repos/taxprofiler/assets/samplesheet_schema.yaml
lrwxrwxrwx 1 cgf8xr cgf8xr    31 Sep 19 10:24 test_pep -> /home/cgf8xr/Downloads/test_pep/
```

When we pass the relative path to `./samplesheet.csv` within the `config.yaml`, Nextflow will look for `samplesheet.csv` in this directory. As you can see, the samplesheet is not here, because it is within `test_pep` directory. To be able to read that, we should update the path from `./samplesheet.csv` to `./test_pep/samplesheet.csv`.

The possible solutions I can see here:
- parse `config.yaml` with Groovy before entering the `CHECK_SAMPLESHEET` workflow, so that we can mount the samplesheet directly to the container (no need for `pep_input_base_dir`)
- update eido so that we can pass addidional parameter that will be joined with path to samplesheet specified in `config.yaml`
- somehow mount all the contents of the directory instead of the directory itself (? not sure if it's possible)
- use absolute paths

I chose the last one because that was the easiest, if you can see any other way how can we fix that then please let me know.